agent/overview.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Secure Tunnels Overview
+sidebarTitle: Overview
+description: Learn about ngrok's secure tunnel agent for exposing local services, connecting devices, and creating secure remote access.
+---
+
+The ngrok agent is a lightweight command-line program that creates secure tunnels from ngrok's global network to your local services, devices, and applications.
+It enables you to expose services behind firewalls and NATs, connect remote devices, and create secure site-to-site connectivity without complex network configuration.
+
+- It runs as a standalone executable with zero runtime dependencies on all major operating systems.
+- It supports HTTP, HTTPS, TCP, and TLS protocols for any application or service.
+- You can run multiple endpoints simultaneously and manage them via configuration files or the CLI.
+- It installs as a native OS service for automatic startup, crash recovery, and background operation.
+
+## Concepts
+
+Here are the core elements you should familiarize yourself with to make the most of ngrok's secure tunnels:
+
+<Columns cols={1}>
+	<Card title="Agent CLI" href="/agent/cli/" horizontal>
+		Use command-line commands to start endpoints, manage configuration, and interact with the ngrok API.
+	</Card>
+	<Card title="Configuration File" href="/agent/config/" horizontal>
+		Define multiple endpoints, customize behavior, and manage complex tunnel configurations with YAML.
+	</Card>
+	<Card title="Authtokens" href="/agent/#authtokens" horizontal>
+		Authenticate your agent with ngrok and create endpoints using credentials scoped to your account.
+	</Card>
+	<Card title="TLS Termination" href="/agent/agent-tls-termination/" horizontal>
+		Secure traffic with end-to-end encryption by terminating TLS at the agent for zero-knowledge security.
+	</Card>
+	<Card title="SSH Reverse Tunnel" href="/agent/ssh-reverse-tunnel-agent/" horizontal>
+		Create secure tunnels using SSH public key authentication for agent connections.
+	</Card>
+</Columns>
+
+## Features
+
+Here are the primary capabilities of ngrok's secure tunnel agent:
```

Comment:
Apologies if I didn't bring this up with the UG overview revamp, but I personally don't usually lke these "intro sentences" for sections like this with a bulleted list. I don't feel super strongly so feel free to ignore, but I do feel like the section stands on its own without a sentence saying what it is

Reply:
I don't have a general stance on this, but this particular case feels like it works without it - intro not necessary